cli/package.json

```diff
@@ -116,7 +116,7 @@
     "cypress": "bin/cypress"
   },
   "engines": {
-    "node": ">=12.0.0"
+    "node": ">=14.0.0"
```

Comment:
Might be worth restricting this to drop node 15 & 17 support as well since they are also at EOL.